src/impl_aarch64.c

```diff
@@ -15,7 +15,7 @@
 #include "cpu_features_macros.h"
 
 #ifdef CPU_FEATURES_ARCH_AARCH64
-#if defined(CPU_FEATURES_OS_LINUX) || defined(CPU_FEATURES_OS_ANDROID)
+#if defined(CPU_FEATURES_OS_LINUX) || defined(CPU_FEATURES_OS_ANDROID) || defined(CPU_FEATURES_OS_MACOS)
```

Comment:
@jwatte, recently added new code layout https://github.com/google/cpu_features/pull/194 to separate platform code, current file `src/impl_linux_or_android_aarch64.c` works with `/proc/cpuinfo` and `/proc/self/auxv` which is only suitable for Linux, so this `defined(CPU_FEATURES_OS_MACOS)` should be in other file `src/impl_aarch64_macos.c`, also a PR https://github.com/google/cpu_features/pull/204 has already been created for this, the only thing is that the PR is big and we will merge it with small PRs